__mocks__/react-freeze.ts

```diff
@@ -0,0 +1,11 @@
+import type {Freeze} from 'react-freeze';
+
+type ReactFreezeMock = {
+    Freeze: typeof Freeze;
+};
+
+const reactFreezeMock: ReactFreezeMock = {
+    Freeze: (props) => props.children as JSX.Element,
```

Comment:
WHy do we need the assertion?

Reply:
Lack of that assertion causes the following error:

```
Type 'ReactNode' is not assignable to type 'Element'.
  Type 'undefined' is not assignable to type 'ReactElement<any, any>'.
```

Alternatively, I could change the file extension to **.tsx** and wrap it in **<></>**, wdyt?

Reply:
Not needed, this way is fine, thanks for explanation